### PR TITLE
Fix switch block, add unique IDs, wrap process return value

### PR DESCRIPTION
**Pull request by KaruroChori**
- Fixed the switch block
- Added unique id for each generated file so that we can run multiple instances
- Fixed the returned value for the process to match the one of the script being run.

### CODE REVIEW

File: bin/bundown.ts [hunk 1]
```diff
@@ -1,108 +1,126 @@
...
+        console.warn(`Unknown language "${block.language}"`);
     }
   }
-  const tmp = `${process.env.HOME}/.bundown/tmp/script.ts`
-  write(tmp, script)
-  await $`bun ${tmp}`
-  process.exit(0)
+  const tmp = `${process.env.HOME}/.bundown/tmp/${crypto.randomUUID()}.ts`;
+  write(tmp, script);
+  let t: ShellOutput;
```
Comment (jrysana) [inline bin/bundown.ts]: Would you mind using a longer and more explanatory name for this `t` variable? I haven't done a perfect job of this myself but I'd like to make sure all of our function/variable names are leaning towards long & intuitive.

File: bin/bundown.ts [hunk 2]
```diff
@@ -1,108 +1,126 @@
...
+  try {
+    t = await $`bun ${tmp}`;
+  } catch (e) {
```
Comment (jrysana) [inline bin/bundown.ts]: Same naming comment for this error variable, maybe something to indicate that it originates from the spawned process? `catch ({somePrefix}Error)`

File: bin/bundown.ts (hunk 1 above)
Comment (KaruroChori) [reply bin/bundown.ts]: :+1: , waiting for the biome file before fixing it. I will also rename `tmp` to be more descriptive like `filename`

File: examples/notebook-1.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Notebook example
```
Comment (jrysana) [inline examples/notebook-1.md]: I think it'd be good if we had some examples documents down the line as a form of docs, but I still want to think a bit more about what those should look like so let's remove this file from this PR for now

File: bin/bundown.ts [hunk 4]
```diff
@@ -108,16 +108,16 @@ try {
         console.warn(`Unknown language "${block.language}"`);
     }
   }
-  const tmp = `${process.env.HOME}/.bundown/tmp/${crypto.randomUUID()}.ts`;
-  write(tmp, script);
-  let t: ShellOutput;
+  const filename = `${process.env.HOME}/.bundown/tmp/${crypto.randomUUID()}.ts`;
+  write(filename, script);
+  let process_ret: ShellOutput;
```
Comment (jrysana) [inline bin/bundown.ts]: Bit of a nitpick but let's keep consistent `camelCase` names for vars/functions, like `processOutput` and `processError` or `processException`

File: format.md [hunk 5]
```diff
@@ -5,5 +5,5 @@ We use `biome` to format our code.
 To format all files in the project, make sure you have `biome` installed and run:
 
 ```bash
-biome format --write .
+bun x biome format --write .
```
Comment (jrysana) [inline format.md:8]: Nice :D

File: yarn.lock [hunk 6]
```diff
@@ -0,0 +1,102 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jrysana) [inline yarn.lock]: Can we uncommit/gitignore `yarn.lock` and `bun.lockb` for now please

**Pull request closed without merging**